tests/test_hostkeys.py

```diff
@@ -33,7 +33,7 @@
 9q/kAYG6sP9f5zuUYP8i7FOFp/6ncCEbbtg/lB+A3iidyxoSWl+9jtoyyDOOVX4UIDV9G11Ml8om3\
 D+jrpI9cycZHqilK0HmxDeCuxbwyMuaCygU9gS2qoRvNLWZk70OpIKSSpBo0Wl3/XUmz9uhc=
 broken.example.com ssh-rsa AAAA
-happy.example.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAIEA8bP1ZA7DCZDB9J0s50l31M\
+happy.example.com  ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAIEA8bP1ZA7DCZDB9J0s50l31M\
```

Comment:
Instead of revising the existing test-case host entries, please create new ones to test these additional whitespace use-cases.

Per my other comment, if a mix of spaces and tabs is acceptable syntax, please add a test case for that, too, after revising the regex.

Reply:
I added a new key to test this behaviour.